app/scripts/controllers/client/CreateClientController.js

```diff
@@ -15,6 +15,63 @@
             scope.forceOffice = null;
             scope.showNonPersonOptions = false;
             scope.clientPersonId = 1;
+            //address
+            scope.addressTypes=[];
+            scope.countryOptions=[];
+            scope.stateOptions=[];
+            scope.addressTypeId={};
+            entityname="ADDRESS";
+            scope.addressArray=[];
+            scope.formData.address=[];
+            //scope.editable=false;
+            //  clientId=routeParams.id;
+
+
+
+
+           /* resourceFactory.clientaddressFields.get(function(data){
+                    scope.addressTypes=data.addressTypeIdOptions;
+                    scope.countryOptions=data.countryIdOptions;
+                    scope.stateOptions=data.stateProvinceIdOptions;
+                }
+            )*/
+            // clientId=sharedVariables.getClientId();
+
+            resourceFactory.addressFieldConfiguration.get({entity:entityname},function(data){
```

Comment:
By default address configuration is false. It is better to check this configuration and call the respective API(s)